### PR TITLE
[ VRF ] Returned t0 configuration for run vrf cases

### LINKED ISSUE #2468: [Vrf] test_vrf fails due to missing attributes in topo_t0.yml
Description: 
After running test_vrf some of test cases fail with such error: 
"AnsibleUndefinedVariable: 'dict object' has no attribute 'podset_number'". 
The reason is that these attributes were deleted from topo_t0.yml by commit #1233 
```
    podset_number: 200
    tor_number: 16
    tor_subnet_number: 2
    max_tor_subnet_number: 16
    tor_subnet_size: 128
    spine_asn: 65534
    leaf_asn_start: 64600
    tor_asn_start: 65100
    failure_rate: 0
```
These attributes is mainly used in test_vrf by vrf_fib.j2

**Steps to reproduce the issue:**
1. Deploy t0 topology.
2. Deploy minigrapgh on DUT
3. Run test_vrf.py 

**Describe the results you received:**
```
E           RunAnsibleModuleFail: run module template failed, Ansible Results =>
E           {"changed": false, "failed": true, "msg": "AnsibleUndefinedVariable: 'dict object' has no attribute 'podset_number'"}

vrf/test_vrf.py::TestVrfIsolation::test_neigh_isolate_vrf1_from_vrf2 ERROR                                                                                       
vrf/test_vrf.py::TestVrfIsolation::test_neigh_isolate_vrf2_from_vrf1 ERROR                                                                                       
vrf/test_vrf.py::TestVrfIsolation::test_fib_isolate_vrf1_from_vrf2 ERROR                                                                                        
vrf/test_vrf.py::TestVrfIsolation::test_fib_isolate_vrf2_from_vrf1 ERROR
```

**Describe the results you expected:**
```
vrf/test_vrf.py::TestVrfIsolation::test_neigh_isolate_vrf1_from_vrf2 PASSED
vrf/test_vrf.py::TestVrfIsolation::test_neigh_isolate_vrf2_from_vrf1 PASSED
vrf/test_vrf.py::TestVrfIsolation::test_fib_isolate_vrf1_from_vrf2 PASSED
vrf/test_vrf.py::TestVrfIsolation::test_fib_isolate_vrf2_from_vrf1 PASSED
```

**Additional information you deem important:**

    **Output of `show version`:**

    ```
    SONiC Software Version: SONiC.master.22-dirty-20201030.044825
    Distribution: Debian 10.6
    Kernel: 4.19.0-9-2-amd64
    Build commit: b132ca09
    ```

    **Attach debug file `sudo generate_dump`:**

    ```
    (paste your output here)
    ```

### PR DESCRIPTION
**Pull request by SavchukRomanLv**
Signed-off-by: Roman Savchuk <romanx.savchuk@intel.com>

### Description of PR

As described in #2538, when run vrf test suite  "AnsibleUndefinedVariable: 'dict object' has no attribute 'podset_number'" appears. The reason is that these attributes were deleted from topo_t0.yml by commit #1233

Summary:
Return back needed configuration for VRF test cases. Fixes #2468

### Type of change

- [+ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

#### What is the motivation for this PR?
Want to have ability to run full  VRF test suite
#### How did you do it?
Returned required configuration of t0 for VRF TC.
#### How did you verify/test it?
Run cases which requires t0 changes:
`
vrf/test_vrf.py::TestVrfIsolation::test_neigh_isolate_vrf1_from_vrf2 PASSED
vrf/test_vrf.py::TestVrfIsolation::test_neigh_isolate_vrf2_from_vrf1 PASSED
vrf/test_vrf.py::TestVrfIsolation::test_fib_isolate_vrf1_from_vrf2 PASSED
vrf/test_vrf.py::TestVrfIsolation::test_fib_isolate_vrf2_from_vrf1 PASSED
`
#### Any platform specific information?
SONiC Software Version: SONiC.master.35-dirty-20201112.031542
Distribution: Debian 10.6
Kernel: 4.19.0-9-2-amd64
Build commit: 6c362a08
Build date: Thu Nov 12 11:49:55 UTC 2020
Platform: x86_64-accton_wedge100bf_32x-r0
HwSKU: montara
ASIC: barefoot

### CODE REVIEW

**Comment by SavchukRomanLv** [comment]
retest vsimage please

**Comment by NazarTkachuk** [comment]
Pls review @wangxin @yxieca

**Comment by wangxin** [comment]
Currently these variables are defined in the fib plugin for announcing routes. If define them again in topo file, there are two sources of these information. However, I feel that topo file is a better place for these definitions. The fib plugin can try to get values from topo file. If no value is in topo file, then the fib plugin can fallback to use default value. So, more changes are needed to make it complete.
* Update the [fib plugin](https://github.com/Azure/sonic-mgmt/blob/master/tests/common/plugins/fib.py)
* Add the definitions to other topo files as well.
Can you make those changes as well?

**Comment by SavchukRomanLv** [comment]
> Currently these variables are defined in the fib plugin for announcing routes. If define them again in topo file, there are two sources of these information. However, I feel that topo file is a better place for these definitions. The fib plugin can try to get values from topo file. If no value is in topo file, then the fib plugin can fallback to use default value. So, more changes are needed to make it complete.
> 
> * Update the [fib plugin](https://github.com/Azure/sonic-mgmt/blob/master/tests/common/plugins/fib.py)
> * Add the definitions to other topo files as well.
> 
> Can you make those changes as well?

@wangxin make changes in latest commit. 
Test it on two topo:
announced 25613 routes on t0, fib TC passed
announced 82877 routes on t01, arp TC passed

File: tests/common/plugins/fib.py [hunk 1]
```diff
@@ -27,7 +27,7 @@ def announce_routes(ptfip, port, routes):
 def generate_routes(family, podset_number, tor_number, tor_subnet_number,
                     spine_asn, leaf_asn_start, tor_asn_start,
                     nexthop, nexthop_v6,
-                    tor_subnet_size = 128, max_tor_subnet_number = 16,
+                    tor_subnet_size, max_tor_subnet_number,
                     router_type = "leaf", tor_index=None):
```
Comment (NazarTkachuk) [inline tests/common/plugins/fib.py:31]: Method `generate_routes` is called in  `sonic-mgmt/tests/decap/test_decap.py`. Pls update it

Comment (SavchukRomanLv) [reply tests/common/plugins/fib.py:31]: Updated method in decap case. Removed hardcoded values. TC tested and passed on t0 topo.

**Review by wangxin — APPROVED** [review]

**Pull request merged**